src/hammer-vlsi/hammer_vlsi/hammer_tool.py

```diff
@@ -95,6 +95,23 @@ def export_config_outputs(self) -> Dict[str, Any]:
         """
         return {}
 
+    def tool_config_prefix(self) -> str:
+        """
+        Returns the config string that contains all tool specific settings
+
+        :return: A string that is the prefix for all tool specific settings
+        """
+        return ""
+
+    def is_newer_version(self, test_version: str) -> bool:
```

Comment:
Hmm, what if instead we had an API that converts the string version into a comparable integer instead? That way maybe tools can do something like 

```python
if version >= self.versionInt("10.0") and version < self.versionInt("13.0"):
  ...
```

Alternatively maybe something like `def versionCompare(self, version_1: str, version_2: str) -> int` where it returns either -1, 0, or 1. What do you think?

Reply:
Sure. I like the first one better. I thought about the second one but it didn't seem needed.

Reply:
The second one is needed if we want to sort a list of version strings though it does seem like a unlikely task. Also we can always implement it using the first so it's more redundant.